src/components/info-general-step/InfoGeneralStep.tsx

```diff
@@ -35,7 +37,25 @@ const UserStepsModal = () => {
     <LanguageStep key='3' />,
     <AddPhotoStep key='4' />
   ]
-
+  const childrenArrStud = [
+    <GeneralInfoStep
+      isUserFetched={isUserFetched}
+      key='1'
+      setIsUserFetched={setIsUserFetched}
+    />,
+    <InterestsStep key='2' />,
+    <LanguageStep key='3' />,
+    <AddPhotoStep key='4' />
+  ]
```

Comment:
Most of this components will "prerender" twice. 
```suggestion
  const childrenArr = [
    <GeneralInfoStep
      isUserFetched={isUserFetched}
      key='1'
      setIsUserFetched={setIsUserFetched}
    />,
     role === 'tutor' ? <SubjectsStep key='2' />:<InterestsStep key='2' />,
    <LanguageStep key='3' />,
    <AddPhotoStep key='4' />
  ]
```